spec/dummy/app/themes/default/assets/stylesheets/layout/_topic-navigation.scss

```diff
@@ -24,11 +24,19 @@
 
 }
 
-.topic-navigation--active {
+%topic-navigation--active {
   border-bottom: 1px solid $action-color;
   margin-bottom: -2px;
 }
 
+#thredded_topics_index .topic-navigation--public {
+  @extend %topic-navigation--active;
+}
+
+#thredded_private_topics_index .topic-navigation--private {
+  @extend %topic-navigation--active;
+}
```

Comment:
@kylefiedler I know the desired method for this is probably to have a class applied on the view itself but what would have required some heavy'ish conditional branching in the view. Instead I reckon leaning on the page's body id is the best approach.

Is this sufficient or is there a mo betta way? School me :)